packages/playground/website/cypress/e2e/blueprints.cy.ts

```diff
@@ -63,8 +63,18 @@ describe('Blueprints', () => {
 	it('enableMultisite step should re-activate the plugins', () => {
 		const blueprint: Blueprint = {
 			landingPage: '/wp-admin/plugins.php',
-			plugins: ['hello-dolly'],
-			steps: [{ step: 'enableMultisite' }],
+			steps: [
+				{ step: 'login' },
```

Comment:
This step started failing because it was missing the login step. 
Without login, the user get's stuck on the login screens instead of being redirected to wp-admin.